content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -0,0 +1,146 @@
+============
+New vehicles
+============
+
+Odoo Fleet manages all vehicles and the accompanying documentation that comes with vehicle
+maintenance and driver's records.
+
+All vehicles are organized on the main :guilabel:`Fleet` dashboard. Each vehicle has its own
+*vehicle form*, which is displayed as a card in the kanban view. Every vehicle form is displayed in
+its current corresponding kanban stage.
+
+To add a new vehicle to the fleet, click the :guilabel:`Create` button, and a blank vehicle form
+will load. Enter the vehicle information in the vehicle form, then click :guilabel:`Save`.
```

Comment:
I see what you mean, but I think it's best to keep the docs for the specific versions, to keep things clear. I do the version specific changes, and I kind of like keeping everything very organized like that. I'll leave this for @StraubCreative to decide since it could go either way.